src/AppBundle/Resources/views/themes/common/parts/author.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if not content.fields.authors.empty %}
```

Comment:
This needs to check if the field exists with `{% if content.fields.authors is defined and not content.fields.authors.empty %}` in order not to crash if mistakenly included in full views for classes which do not have the field. Same goes for other parts templates.

Reply:
I agree, it would be good to add hasField test in all parts templates, just to be sure.

Reply:
Agreed, fixed.